### DISCUSSION TITLE
Where are my benefits?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#143174] [GitHub Education] [answered]
### Hi everyone,

Hello everyone, I have waited for 3 days for my benefits, where are they?

![image](https://github.com/user-attachments/assets/3815d7c7-f023-4fcc-95ab-490fec3f962a)

**Comment by ComputingVictor** [comment, marked as answer]
Hi @Anshkabra2012!! You can check it out the different benefits here: [https://education.github.com/pack](https://education.github.com/pack). Also there, you can check the description of each one.

Take into account that the process to activate your account to student one, could take within 7 days aprox. That was my case. You will receive a mail and the status of approved will change to purple once the request is fully completed.

<img width="97" alt="Captura de pantalla 2024-10-31 a las 13 42 01" src="https://github.com/user-attachments/assets/1aa83ada-2f31-40c0-829f-0dd43e0f90eb">

**Reply by ghost** [reply]
ok,thanks!

**Reply by ComputingVictor** [reply]
@Anshkabra2012  If it helped you, could you mark it as an answer?

Many Thanks!

**Comment by joegeorge022** [comment]
# 🎓 GitHub Education Benefits for Students 🎓

GitHub Education provides incredible resources and perks to help students succeed. Here are some highlights:

---

## 📚 Free Developer Tools
   - Access the **Student Developer Pack** with over 100 free or discounted tools for coding, design, cloud services, and more.

## 🚀 Project Collaboration
   - Collaborate on **real-world projects** with peers, enhancing teamwork and version control skills.

## 🌐 Cloud Computing Credits
   - Enjoy free credits for platforms like **Azure**, **AWS Educate**, and **DigitalOcean**.

## 📊 Build Your Portfolio
   - Showcase your projects to potential employers on GitHub, demonstrating coding and project experience.

## 🎉 Community & Networking
   - Connect with peers globally, join hackathons, and expand your network through GitHub's student community.

## 🛠️ Support for Open Source
   - Contribute to open-source projects and learn best practices in a real-world environment.

---

[Click here to learn more](https://github.com/education/students)

---

**Comment by ghost** [comment]
Thanks everyone I got my benefits long back in October, really good benefits I must say though 😁

**Reply by joegeorge022** [reply]
Glad you got it✨
